src/Ledger/Conway/Epoch.lagda

```diff
@@ -406,27 +414,25 @@ its results by carrying out each of the following tasks.
       trWithdrawals   = esW .withdrawals
       totWithdrawals  = ∑[ x ← trWithdrawals ] x
 
-      retired    = (pState .retiring) ⁻¹ e
       payout     = govActionReturns ∪⁺ trWithdrawals
       refunds    = pullbackMap payout toRwdAddr (dom (dState .rewards))
       unclaimed  = getCoin payout - getCoin refunds
 
       govSt' = filter (λ x → proj₁ x ∉ mapˢ proj₁ removed') govSt
 
-      dState' = ⟦ dState .voteDelegs , dState .stakeDelegs ,  dState .rewards ∪⁺ refunds ⟧
-
-      pState' = ⟦ pState .pools ∣ retired ᶜ , pState .retiring ∣ retired ᶜ ⟧
+      dState'' = record dState' { rewards =  dState' .rewards ∪⁺ refunds }
 
       gState' = ⟦ (if null govSt' then mapValues (1 +_) (gState .dreps) else (gState .dreps))
                 , gState .ccHotKeys ∣ ccCreds (es .cc) ⟧
 
       certState' : CertState
-      certState' = ⟦ dState' , pState' , gState' ⟧
+      certState' = ⟦ dState'' , pState' , gState' ⟧
+
+      utxoSt'' = ⟦ utxoSt' .utxo , utxoSt' .fees , utxoSt' .deposits ∣ mapˢ (proj₁ ∘ proj₂) removedGovActions ᶜ , 0 ⟧
```

Comment:
Before adding `POOLREAP`, `mkStakeDistrs` was fed the deposits including those of retiring pools (in the environment of the `RATIFIES` transition). The `pools` of `RATIFIES` also included retiring pools.

This PR is changing `mkStakeDistrs` by excluding the deposits of retiring replicas. This was accidental, and I guess I must revert that. This in turn raises the question of whether `POOLREAP` should happen after removing GovActions from deposits. At the moment `POOLREAP` is done before for no particular reason that I'm aware of.

Does it make sense?